docs/developer-docs/latest/update-migration-guides/migration-guides/v4/plugin-migration.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Plugin migration guide for v4 - Strapi Developer Docs
+description: Migrate your plugins from v3.6.8 to v4.0 with step-by-step instructions
+sidebarDepth: 2
+canonicalUrl:
+---
+
+<!-- TODO: update SEO -->
+
+# v4 plugin migration guide
+
+The goal of this guide is to get a v3 plugin up and running on v4 as fast as possible by resolving breaking changes. It is not an exhaustive resource for the v4 plugin APIs, which are described in the [Server API](/developer-docs/latest/developer-resources/plugin-api-reference/server.md#server-api-for-plugins) and [Admin Panel API](/developer-docs/latest/developer-resources/plugin-api-reference/admin-panel.md#admin-panel-api-for-plugins) documentations.
+
+Migrating a plugin from Strapi v3.6.9 to v4.0.4 consists in:
+
+- [enabling the plugin](/developer-docs/latest/update-migration-guides/migration-guides/v4/plugin/enable-plugin.md)
+- [updating the folder structure](/developer-docs/latest/update-migration-guides/migration-guides/v4/plugin/update-folder-structure.md)
+- [migrating the back end](/developer-docs/latest/update-migration-guides/migration-guides/v4/plugin/migrate-back-end.md)
```

Comment:
Should we say `migrating the back end code` ? or `updating the backend api` ? Or something else ?
I find that `migrating the back end` is not as specific as the 2 dots above

Reply:
Let's go for "back-end code" and "front-end code", thanks, Pierre!